apps/webcert/src/hooks/useGoBackEffect.ts

```diff
@@ -0,0 +1,22 @@
+import { useHistory } from 'react-router'
+import { useAppSelector } from '../store/store'
+import { getPatientId } from '../store/srs/srsSelectors'
+
+export const useGoBackEffect = () => {
```

Comment:
"Effect" is reserved for hooks that utilize "useEffect" to sync with external systems. Since this hook is not using that, we can remove that part from the name.